pandas/tests/frame/test_json.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
no, this can go (just the test), don't add a new file in ``pandas/io/tests/json/test_json_norm.py``